### PR TITLE
feat(dedicated.server): add ola guides

### PR DESCRIPTION
**Pull request by lizardK**
# Dedicated server OLA guide

## Description of changes

Add a "How To guide" for OLA feature

## References
MANAGER-3903

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
   "dedicated_server_interfaces_title": "Réseau",
+  "dedicated_server_interfaces_guides": "Guides",
+  "dedicated_server_interfaces_guides_ola": "How to configure OLA?",
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/translations/Messages_fr_FR.json]: French sentence maybe?

File: packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html [hunk 2]
```diff
@@ -1,7 +1,16 @@
 <div data-ovh-alert></div>
 
-<h1 data-translate="dedicated_server_interfaces_title"></h1>
+<div class="row">
+    <h1 data-translate="dedicated_server_interfaces_title" class="col-auto mr-auto"></h1>
+    <oui-guide-menu text="{{::'dedicated_server_interfaces_guides' | translate}}" class="col-auto px-5">
+        <oui-guide-menu-item  href="https://docs.ovh.com/gb/en/dedicated/ola-manager/" external>
```
Comment (frenautvh) [inline packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html]: can we put the localized guide in app/config/constants.config.js ?

Comment (antleblanc) [reply packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html]: It's only used in one place and not supposed to be translated (not yet).

@lizardK correct me if I'm wrong.

Comment (lizardK) [reply packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html]: Correct. There's only EN version and it won't be translated.

File: packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html [hunk 3]
```diff
@@ -1,7 +1,16 @@
 <div data-ovh-alert></div>
 
-<h1 data-translate="dedicated_server_interfaces_title"></h1>
+<div class="row">
+    <h1 data-translate="dedicated_server_interfaces_title" class="col-auto mr-auto"></h1>
```
Comment (jleveugle) [inline packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html]: `.row`, the child sould be a `div.col-*`

Comment (antleblanc) [reply packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html]: @jleveugle is already the case (see: https://getbootstrap.com/docs/4.3/layout/grid/#margin-utilities)

Comment (lizardK) [reply packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html]: It comes from bs4 doc : 

```
  <div class="row">
    <div class="col-auto mr-auto">.col-auto .mr-auto</div>
    <div class="col-auto">.col-auto</div>
  </div>
```

File: packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html [hunk 4]
```diff
@@ -1,7 +1,16 @@
...
+    <oui-guide-menu text="{{::'dedicated_server_interfaces_guides' | translate}}" class="col-auto px-5">
+        <oui-guide-menu-item data-href="https://docs.ovh.com/gb/en/dedicated/ola-manager/" external>
```
Comment (frenautvh) [inline packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html]: url to be defined in localized app constants ? guides.url[subsidiary]

Comment (antleblanc) [reply packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html]: @frenautvh please refer to my previous comment :)

https://github.com/ovh/manager/pull/1719#discussion_r342973181

**Pull request merged**